mne/viz/_3d.py

```diff
@@ -1277,28 +1276,16 @@ def _plot_glyphs(
     check_inside=None,
     nearest=None,
 ):
+    from matplotlib.colors import ListedColormap, to_rgba
+
+    _validate_type(mark_inside, bool, "mark_inside")
     if surf is not None and len(loc) > 0:
         defaults = DEFAULTS["coreg"]
         scalars, vectors, proj_pts = _orient_glyphs(
             loc, surf, project_points, mark_inside, check_inside, nearest
         )
         if mark_inside:
-            from matplotlib.colors import ListedColormap
-
-            color = np.append(color, 1)
-            colormap = ListedColormap(
-                np.array(
-                    [
-                        (
-                            0,
-                            0,
-                            0,
-                            1,
-                        ),
-                        color,
-                    ]
-                )
-            )
```

Comment:
so was the ultimate problem that this `(0, 0, 0, 1)` array should have been `(0, 0, 0, 255)` (or `(0., 0., 0., 1.)`)?

Reply:
No that's just a cleanup. I think it was that we were using `int` as the dtype for `scalars` rather than `float`